developer/general/how-to-edit-the-rst-documentation.rst

```diff
@@ -306,149 +306,195 @@ It is recommended to use a virtual environment, f.ex.
 
-    .. code-block:: console
+2.  **Install Sphinx and Required Extensions**
 
-     $ sphinx-build --version
+   .. code-block:: console
 
+      (.venv) $ pip install -r requirements.txt
+      (.venv) $ pip install sphinx-lint sphinx-autobuild
 
-4.  **Build Documentation**
 
-   Use `sphinx-build` with the `-v` (verbose) flag to generate detailed output during the build process.
+Linting the documentation
+^^^^^^^^^^^^^^^^^^^^^^^^^
 
-    .. code-block:: console
 
-     $ sphinx-build -v -b html qubes-doc _build/html
-
-
-   The build command specifies the source directory (:file:`qubes-doc`), the output directory (:file:`_build/html`), and the builder (`html`)
-   and will process all source files in the :file:`qubes-doc` directory,
-   generate HTML output in the :file:`_build/html` directory, and print detailed build information to the console.
-   Pay attention to errors and warning in the output!
-   Please do not introduce any new warnings and fix all errors.
-
-5.  **Run Linting**
+1.  **Run Linting**
 
    The `sphinx-lint` extension checks for common issues like missing references, invalid directives,
    or formatting errors.
+   The `sphinx-lint` extension checks for common issues like missing references, invalid directives,
+   or formatting errors. Run the linting step using the :program:`sphinx-lint` command.
 
-    .. code-block:: console
+   .. code-block:: console
 
-     $ sphinx-lint qubes-doc
+      (.venv) $ sphinx-lint -i .venv .
 
 
-6.  **Run Link Checking**
+2.  **Run Link Checker**
 
    The `sphinx-linkcheck` extension verifies the validity of all external and internal links.
 
    The results will be written to the :file:`_build/linkcheck` directory with a detailed report in :file:`output.txt` or :file:`output.json` files
-   of all checked links and their status (e.g., OK, broken).
+   of all checked links and their status (e.g., OK, broken, timeout).
+
+   .. code-block:: console
+
+      (.venv) $ sphinx-build -b linkcheck . _build/linkcheck
 
-    .. code-block:: console
 
-     $ sphinx-build -b linkcheck qubes-doc _build/linkcheck
+Building the documentation
+^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 
-7.  **Use sphinx-autobuild for development**
+-  **Build Documentation**
+
+   Use `sphinx-build` with the `-v` (verbose) flag to generate detailed output during the build process.
+   The build command specifies the source directory (current directory :file:`.`, :file:`qubes-doc` in this case), the output directory (:file:`_build/html`), and the builder (`html`).
+
+   .. code-block:: console
+
+      (.venv) $ sphinx-build -v -b html . _build/html
+
+
+   The build command specifies the source directory (:file:`qubes-doc`), the output directory (:file:`_build/html`), and the builder (`html`)
+   and will process all source files in the :file:`qubes-doc` directory,
+   generate HTML output in the :file:`_build/html` directory, and print detailed build information to the console.
```

Comment:
`The build command specifies the source directory (:file:`qubes-doc`), the output directory (:file:`_build/html`), and the builder (`html`) and will process ...` should be `The build command will process ...`